src/components/tables/Captures/useCaptureColumns.tsx

```diff
@@ -21,34 +32,8 @@ const defaultColumns: ColumnProps[] = [
     },
 ];
 
-const statsHeader: ColumnProps = {
-    field: null,
-    cols: 2,
-    renderHeader: (index, selectableTableStoreName) => {
-        return (
-            <StatsHeader
-                key={`captures-statsHeader-${index}`}
-                headerSuffix="data.written"
-                selectableTableStoreName={selectableTableStoreName}
-            />
-        );
-    },
-};
-
 const useCaptureColumns = (): ColumnProps[] => {
-    const tenantDetails = useTenantDetails();
-
-    const hasDetails = useMemo(() => hasLength(tenantDetails), [tenantDetails]);
-
-    return useMemo(() => {
-        if (hasDetails) {
-            const response = [...defaultColumns];
-            response.splice(3, 0, statsHeader);
-            return response;
-        } else {
-            return defaultColumns;
-        }
-    }, [hasDetails]);
+    return useMemo(() => defaultColumns, []);
```

Comment:
This could just be shared through a `const` but leaving the hook in to reduce scope a bit and in case we need to end up undoing this in the future.